sisl/orbital.py

```diff
@@ -1130,3 +1130,50 @@ def __setstate__(self, d):
             self.__init__(d["name"], q0=d["q0"], tag=d["tag"])
         else:
             self.__init__(d["name"], (d["r"], d["f"]), q0=d["q0"], tag=d["tag"])
+
+
+@set_module("sisl")
+class HydrogenicOrbital(AtomicOrbital):
+    r""" A hydrogen-like atomic orbital defined by an effective atomic number Z
+    in addition to the usual quantum numbers (n, l, m).
+    The returned orbital is properly normalized.
+
+    See https://en.wikipedia.org/wiki/Hydrogen-like_atom
+
+    Parameters
+    ----------
+    Zeff : float
+        effective atomic number
+    n : int
+        principal quantum number
+    l : int
+        angular momentum quantum number
+    m : int
+        magnetic quantum number
+    R : float, optional
+        max range of the constructed orbital
+
+    Examples
+    --------
+    >>> carbon_pz = HydrogenicOrbital(3.2, 2, 1, 0)
+
+    """
+
+    def __init__(self, Zeff, n, l, m, *args, **kwargs):
+
+        self._Zeff = Zeff
+
+        R = kwargs.get("R", 10.)
+        r = np.linspace(0, R, 1000)
+        a0 = 0.529177 # Bohr radius
```

Comment:
I have added the Bohr radius in the constant module, could you use `a0("Ang")`? This should give the same value (but with more significant digits).

Reply:
Done